### ISSUE TITLE
Empty revive dashboard (0.8.6)

### ISSUE THREAD

**Issue by jpfroud** [LibreVR/Revive#228]
Hi,

here's the look of my dashboard:
http://i.imgur.com/nWaR32a.png

I tried these but the game isn't showing:

- I first installed oculus software folder into F drive, I uninstalled it (removed appData folders) and reinstalled into C drive
- Did the same with revive installation, F: then C:
- Deleted and downloaded the game again

But the dashboard remains desperately empty.

**Comment by jpfroud** [comment]
After a reboot (windows, not steamVR) I can see the game in the dashboard but it crashes (I had to use the vive controller to launch the game, the xbox didn't do anything)

http://i.imgur.com/8QNByp8.png

**Comment by LibreVR** [comment]
Can you send me that folder mentioned in the error message? It may contain some useful information to debug the problem.

**Comment by jpfroud** [comment]
I don't have it anymore. Since I thought the solution to the first issue (empty dashboard) was to restart, I uninstalled all (oculus home, games and revive in C:) and reinstalled them all into F:

But even after a restart, I'm back to the first issue, the dashboard is empty.

Looks like there's an issue depending on the path.

**Comment by jpfroud** [comment]
I solved the empty dashboard issue by creating a symbolic link from C: to F:

`mklink /J "C:\Program Files\Oculus\Software" "F:\Program Files\Oculus\Software"`

(I first had to delete `C:\Program Files\Oculus\Software`)

Now I'm back to the second issue, here's the folder
[2016-08-15_175230.zip](https://github.com/LibreVR/Revive/files/418396/2016-08-15_175230.zip)

**Comment by jpfroud** [comment]
And found the answer to the second issue here: https://github.com/LibreVR/Revive/issues/224

I had to opt-out of SteamVR beta. Now everything looks good.

**Comment by LibreVR** [comment]
> I solved the empty dashboard issue by creating a symbolic link from C: to F:

That's strange, this should be detected from the registry key. Did you move the Oculus folder to the other drive manually at any point? If you place it on your F drive through the setup this should be detected by Revive.

**Comment by jpfroud** [comment]
No I didn't do any manual move. But during the oculus setup, I couldn't change this location to F:
`C:\Program Files\Oculus`

I could only change the game installation folder:
`F:\Program Files\Oculus\Software`

At the end of the setup, you have 2 "Software" folders:
`C:\Program Files\Oculus\Software`
`F:\Program Files\Oculus\Software`

Maybe I didn't have to create a link and just had to delete this folder:
`C:\Program Files\Oculus\Software`

**Comment by LibreVR** [comment]
> I could only change the game installation folder

That's interesting, I didn't know you could do that separately.

**Comment by supermanscottbristol** [comment]
I'm getting an empty dashboard after a full reinstall of windows.

Overlay log says :

libpng warning: iCCP: known incorrect sRGB profile
Saved manifest to: V:\Program Files\Revive\revive.vrmanifest
Runtime directory: file:///V:/Program Files (x86)/Steam/steamapps/common/SteamVR/
Oculus directory: file:///C:/Program Files/Oculus/

**Comment by supermanscottbristol** [comment]
Just as an update - tried with .8.6 and .8.5 and same thing.

Put the link in from V: to C:\Program Files\Oculus\Software and it now works.

**Comment by higate** [comment]
I have exact same problem. If I install the "Software" folder on C: under the Oculus folder everything works. If split it, so the Software folder is on F: and the Oculus on C:, then I get a blank TAB :-(

**Comment by supermanscottbristol** [comment]
higate - as above, use the link solution.

**Comment by LibreVR** [comment]
I'll fix this problem in the next release.

**Comment by higate** [comment]
sorry missed that one... will try, 
LibreVR glad to hear ;-) Keep up the good work.

**Comment by sbawden77** [comment]
I've been trying everything that is listed on this page for the empty revive dashboard. Nothing works. So games like lucky tale, I can get to load using the injector. I unfortunately bought eve valkyrie, and have tried everything. Nothing will get it to load. I've Uninstalled revive, oculus home, steam vr, and then reinstalled them with the current versions. Nothing. No games on the dashboard, and eve says I needs to use a launcher. If I drag the injector onto eve.exe it brings up a screen saying connecting to cloud. Then nothing happens. Any ideas? I'm at a loss.

**Comment by supermanscottbristol** [comment]
Did you definitely set up the symbolic link correctly?

**Comment by sbawden77** [comment]
I erased the software folder under the c drive, then linked the D drive oculus/software folder to c: program files/ oculus. Did i do that right?

**Comment by supermanscottbristol** [comment]
If when you go into c:\program files\oculus it now looks like there's a software folder there, then yes that shouldve fixed it.

**Comment by sbawden77** [comment]
Ok. So does it matter where revive is installed? I'll double check the oculus folder tonight when I get off work. Thanks for your help.

**Comment by supermanscottbristol** [comment]
Don't think it matters.  I've installed mine on a V: drive.

**Comment by sbawden77** [comment]
Ok

**Comment by sbawden77** [comment]
So the software folder was there, but still nothing. So I completely erased oculus home and just put everything into my c drive. Games directory too. Then reinstalled steam vr. Now it works. I don't know why it wasn't working before, and that wasn't where I wanted the game directory, but good enough. Thanks for your help.

**Comment by LibreVR** [comment]
The next version will support the new software library feature in Oculus Home. For now I recommend installing Oculus Home and its software library into the same folder or else Revive will not detect them.

**Comment by eikonoklastes** [comment]
Hello,

I'm still getting a blank screen after linking my directories. My log files shows this:

```
libpng warning: iCCP: known incorrect sRGB profile
Loaded manifest from: C:\Program Files\Revive\revive.vrmanifest
Runtime directory: file:///C:/Program Files (x86)/Steam/steamapps/common/SteamVR/
Oculus directory: file:///C:/Program Files/Oculus/
Loading assets manifest: file:///c:/Program Files/Revive/Qt
Loading assets manifest: file:///c:/Program Files/Revive/QtGraphicalEffects
Loading assets manifest: file:///c:/Program Files/Revive/QtMultimedia
Loading assets manifest: file:///c:/Program Files/Revive/QtQml
Loading assets manifest: file:///c:/Program Files/Revive/QtQuick
Loading assets manifest: file:///c:/Program Files/Revive/QtQuick.2
Loading assets manifest: file:///c:/Program Files/Revive/Revive
Loading assets manifest: file:///c:/Program Files/Revive/audio
Loading assets manifest: file:///c:/Program Files/Revive/bearer
Loading assets manifest: file:///c:/Program Files/Revive/iconengines
Loading assets manifest: file:///c:/Program Files/Revive/imageformats
Loading assets manifest: file:///c:/Program Files/Revive/mediaservice
Loading assets manifest: file:///c:/Program Files/Revive/platforms
Loading assets manifest: file:///c:/Program Files/Revive/playlistformats
Loading assets manifest: file:///c:/Program Files/Revive/qmltooling
Loading assets manifest: file:///c:/Program Files/Revive/translations
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
```

I couldn't find any info on how to fix the "Parse error"

I guess I'll just reinstall everything onto my C: until the next version is out.

Thank you.

**Comment by nateev** [comment]
Hello, i have the exact same errors as eikonoklastes

libpng warning: iCCP: known incorrect sRGB profile
Loaded manifest from: C:\Program Files\Revive\revive.vrmanifest
Runtime directory: file:///J:/jeux/steam/steamapps/common/OpenVR/
Oculus directory: file:///C:/Program Files/Oculus/
Loading assets manifest: file:///c:/Program Files/Revive/Qt
Loading assets manifest: file:///c:/Program Files/Revive/QtGraphicalEffects
Loading assets manifest: file:///c:/Program Files/Revive/QtMultimedia
Loading assets manifest: file:///c:/Program Files/Revive/QtQml
Loading assets manifest: file:///c:/Program Files/Revive/QtQuick
Loading assets manifest: file:///c:/Program Files/Revive/QtQuick.2
Loading assets manifest: file:///c:/Program Files/Revive/Revive
Loading assets manifest: file:///c:/Program Files/Revive/audio
Loading assets manifest: file:///c:/Program Files/Revive/bearer
Loading assets manifest: file:///c:/Program Files/Revive/iconengines
Loading assets manifest: file:///c:/Program Files/Revive/imageformats
Loading assets manifest: file:///c:/Program Files/Revive/mediaservice
Loading assets manifest: file:///c:/Program Files/Revive/platforms
Loading assets manifest: file:///c:/Program Files/Revive/playlistformats
Loading assets manifest: file:///c:/Program Files/Revive/qmltooling
Loading assets manifest: file:///c:/Program Files/Revive/translations
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error
qrc:/Oculus.js:86: JSON.parse: Parse error

**Comment by LibreVR** [comment]
That's very strange, it's trying to load asset manifests from the Revive directory, that should never happen. In any case this will be fixed in the next release.

**Comment by LibreVR** [comment]
Fixed in commit 32099da4f4fcfc2b9e031b05b865200da97c9cf3.

**Comment by LibreVR** [comment]
This should be fixed in [the latest release](https://github.com/LibreVR/Revive/releases/tag/0.9), please test it.
